core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -29,34 +29,37 @@ SERVER_LOG_FILE=$(grep -Po "(?<=^node.server-log-file=).*" $NODE_PROPERTIES | ta
 LAUNCHER_LOG_FILE=$(grep -Po "(?<=^node.launcher-log-file=).*" $NODE_PROPERTIES | tail -n 1)
 CONFIGURATION=(--launcher-config /usr/lib/trino/bin/launcher.properties --data-dir "$DATA_DIR" --node-config "$NODE_PROPERTIES" --jvm-config /etc/trino/jvm.config --config /etc/trino/config.properties --launcher-log-file "${LAUNCHER_LOG_FILE:-/var/log/trino/launcher.log}" --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}")
 
+set -a
```

Comment:
maybe instead require env.sh to use `export V=...`

Reply:
Refactored into an associative array